doc/apidoc/smac.epm.rfr_imputator.rst

```diff
@@ -0,0 +1,7 @@
+smac\.epm\.rfr\_imputator module
+================================
+
+.. automodule:: smac.epm.rfr_imputator
```

Comment:
@KEggensperger could you please add the return values to the docstring of the `impute()` function?